proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -82,6 +82,8 @@ service TerminalService {
 
   // GetKubes returns filtered, sorted, and paginated kubes
   rpc GetKubes(GetKubesRequest) returns (GetKubesResponse);
+  // GetApps returns a filtered and paginated list of apps.
+  rpc GetApps(GetAppsRequest) returns (GetAppsResponse);
```

Comment:
In case of the app search, I could use the RPC for unified resources. 
However, I will need the apps RPC for access requests. We need to show `totalCount` there, and it is not returned for unified resources.
For this reason, I thought I would use it here as well and just copy the existing patterns (for servers/kubes/dbs).

Reply:
This makes me wonder how hard it'd be to rip out the existing search implementation in `ResourcesService.searchResources` and replace it with a call to the unified resources RPC so that we make a single call per cluster and not four.

But between this and opening HTTP apps from within Connect, I think opening those apps is a better use of our time.